### ISSUE TITLE
vscode://... protocol failed on Linux

### ISSUE THREAD

**Issue by e-zz** [microsoft/vscode#197541]
Does this issue occur when all extensions are disabled?: Yes/No

- Version: 1.84.0
- OS: Linux x64 6.2.0-36-generic (Ubuntu 22.04)

Steps to Reproduce:
To open any local file, let's say `~/e-zz/WorkDir/test.txt`
1. run `xdg-open /vscode://file//home/e-zz/WorkDir/test.txt`
2. nothing will happen

While with the previous version (vscode 1.83.1), above steps will invoke vscode to open it.

**Comment by zjffun** [comment]
```diff
- vscode://file//home/e-zz/WorkDir/test.txt
+ vscode://file/home/e-zz/WorkDir/test.txt
```

Same issue, and I find one `/` after file worked.

**Comment by zjffun** [comment]
It seems fixed in #197511

**Comment by e-zz** [comment]
> It seems fixed in #197511

Thanks. Indeed it's fixed now in VSCode 1.84.1

**Comment by inmanturbo** [comment]
@e-zz 
This still seems to be broken in VSCode 1.84.2 using the snap on ubuntu 22.04

➜  ~ lsb_release -a                                           
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy

➜  ~ code --version
1.84.2
1a5daa3a0231a0fbba4f14db7ec463cf99d7768e
x64
